rpi3.mk

```diff
@@ -192,6 +192,7 @@ endif
 .PHONY: rpi3-firmware-clean
 rpi3-firmware-clean:
 	rm -f $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT)
+	rm -rf $(RPI3_STOCK_FW_PATH)
```

Comment:
@igoropaniuk maybe you can remove each file one by one and then the directory if it's empty, as in https://github.com/jforissier/optee_client/commit/6432f5194822d7da586d736e8733d7b78401bdec